recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "plotly" %}
-{% set version = "5.6.0" %}
+{% set version = "5.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d86e44ebde38f4753dff982ab9b5e03cf872aab8fdf53a403e999ed378154331
+  sha256: b0536e72bbc0b3cf169ac1fd00759d77aae7bb12ae378cdc75c5dc362f5de576
 
 build:
   number: 0
```

Comment:
We should remove noarch: python and build it as arch-specific:
- Add skip: `True. # [py<36]`
- Use plain `python` in `run` without pinning
-  Update test/imports https://github.com/plotly/plotly.py/blob/ddd5ba6ba21f176914da2ee448331e05eaebb471/packages/python/plotly/setup.py#L519-L534
- Remove `six` from `run` https://github.com/plotly/plotly.py/blob/ddd5ba6ba21f176914da2ee448331e05eaebb471/packages/python/plotly/setup.py#L556 and https://github.com/plotly/plotly.py/blob/v5.9.0/CHANGELOG.md#fixed-1

See:
1. https://github.com/plotly/plotly.py/blob/v5.9.0/packages/python/plotly/pyproject.toml (but do not add jupyterlab)